README.md

```diff
@@ -102,18 +102,18 @@ Installing
 
 ### Configuring
 
-Default configuration file is located in `src/environments/` folder.
+Default configuration file is located in `config/` folder.
 
-To change the default configuration values, create local files that override the parameters you need to change. You can use `environment.template.ts` as a starting point.
+To override the default configuration values, create local files that override the parameters you need to change. You can use `config.example.yml` as a starting point.
 
--	Create a new `environment.dev.ts` file in `src/environments/` for a `development` environment;
--	Create a new `environment.prod.ts` file in `src/environments/` for a `production` environment;
+-	Create a new `config.(dev or development).yml` file in `config/` for a `development` environment;
+-	Create a new `config.(prod or production).yml` file in `config/` for a `production` environment;
 
-The server settings can also be overwritten using an environment file.
+The settings can also be overwritten using an environment file or environment variables.
 
 This file should be called `.env` and be placed in the project root.
 
-The following settings can be overwritten in this file:
+The following non-convention settings:
```

Comment:
If all our env variables are prefixed with `DSPACE_`, then they'd align more with these older settings...making this no longer "non-convention settings".  Instead, we'd just need to update this list to be: `DSPACE_UI_HOST`, `DSPACE_UI_PORT`, etc... and all the `DSPACE_REST_*` env variables would remain the same.

Reply:
Noted.